odinmain/src/main/kotlin/me/odinmain/utils/skyblock/dungeon/DungeonEnums.kt

```diff
@@ -23,6 +23,37 @@ data class DungeonPlayer(
     var isDead: Boolean = false
 )
 
+data class Puzzle(
+    val name: String,
+    var status: PuzzleStatus? = null
+) {
+    companion object {
```

Comment:
this could also be an enum, then you wouldnt need the list

Reply:
Im not entirely sure enums can have duplicate values, and both of these need to be able to have duplicate values. Both need to be able to handle duplicate different values.